PowerEditor/src/WinControls/Preference/preference.rc

```diff
@@ -126,6 +126,7 @@ BEGIN
     CONTROL         "Enable Notepad++ auto-updater",IDC_CHECK_AUTOUPDATE, "Button",BS_AUTOCHECKBOX | BS_MULTILINE | WS_TABSTOP,37,143,217,10
     CONTROL         "Don't fill find field in Find dialog with selected word", IDC_CHECK_STOPFILLINGFINDFIELD, "Button",BS_AUTOCHECKBOX | BS_MULTILINE | WS_TABSTOP,37,158,217,10
     CONTROL         "Use Monospaced font in Find dialog (Need to restart Notepad++)",IDC_CHECK_MONOSPACEDFONT_FINDDLG, "Button",BS_AUTOCHECKBOX | WS_TABSTOP,37,173,325,10
+    CONTROL         "Use DirectWrite technology (Need to restart Notepad++)",IDC_CHECK_DIRECTWRITE_ENABLE, "Button",BS_AUTOCHECKBOX | WS_TABSTOP,37,188,325,10

```

Comment:
Can we find another term more user friendly than "Use DirectWrite technology"?

Reply:
IMHO we should mention DirectWrite explicitly anyway. Maybe `Render text using DirectWrite (may be beneficial for performance and quality)`

Reply:
Actually, I don't know a better name. But I'm open for suggestions.

I agree with @KOLANICH, that we should display, what we are doing, and not something obscure. But I also worry about the length of the name a little.

So my second thought was `Use DirectWrite API (Need to restart Notepad++)`.

Reply:
"Render text using DirectWrite (may be beneficial for performance and quality)" sounds better.
Agree.

Reply:
@Uhf7 said:  "I ... worry about the length of the name a little."

@donho suggests: _Render text using DirectWrite (may be beneficial for performance and quality)_

Conclusion:  Don's not worried about the length!  :-)

Is it reasonable to do?: 
_Render text using DirectWrite_ 
and then as a hover tooltip?:
_May be beneficial for performance and quality; need to restart Notepad++_

Reply:
Good idea @sasumner !
What do you think @Uhf7 ?

Reply:
Maybe it's better, for aligning with `Use Monospace font in Find dialog (need to restart Notepad++)`:
```
Render text using DirectWrite (need to restart Notepad++)
and then as a hover tooltip?:
May be beneficial for performance and quality
```

Reply:
A downside is that it creates two separate things to language translate.

Reply:
@sasumner  It's a very little sacrifice if it's more clear for the users :)

Reply:
I don't find many tooltips in the preferences menu. Could this be the 1st one? And if, do we really want to start this? I could create a 2nd line, for example, instead of the tooltip.

Reply:
Better text: _Use DirectWrite (May improve rendering special characters, need to restart Notepad++)_. Improving performance is a too vague promise perhaps.

Reply:
@Uhf7  OK, sounds good enough.
Please make one line label - We have enough room. and please make it as long as possible even in English it doesn't take all room: the translation takes more room than the original text generally.